packages/inertia/src/inertia.js

```diff
@@ -116,6 +118,8 @@ export default {
           page.props = { ...this.page.props, ...page.props }
         }
 
+        this.saveScrollPositions()
```

Comment:
Curious, what was the benefit/purpose of moving this from line 74?

Reply:
This way, you can still scroll while a page is loading (e.g. on a slow connection)